app/pages/angelfish.tsx

```diff
@@ -4,29 +4,21 @@ import Head from "next/head";
 import { Progress } from "../components/Progress";
 import { DosesDescriptionList } from "../components/DosesDescriptionList";
 import { DarkModeToggle } from "../components/DarkModeToggle";
-import { DhbPopulationDoseData } from "../types/VaccineDataTypes";
-import fetchHomePageProps from "../services/homePagePropsService";
+import { DhbPopulationDoseDataWithRegion } from "../types/VaccineDataTypes";
+import fetchHomePageProps from "../services/angelfishPagePropsService";
 import { useHasMounted } from "../hooks/useIsMounted";
 import { RegionDropdown } from "../components/RegionDropdown";
-import { DhbRegionId } from "../components/RegionDropdown/RegionDropdown";
+import { DhbRegionId } from "../types/VaccineDataTypes";
 import { useState } from "react";
 
 export type HomePageProps = {
-  allDhbsPopulationDoseData: DhbPopulationDoseData[];
-  aucklandDhbsPopulationDoseData: DhbPopulationDoseData[];
-  combinedAucklandDhbsPopulationDoseData: DhbPopulationDoseData;
+  allDhbsPopulationDoseData: DhbPopulationDoseDataWithRegion[];
   dataValidAsAtTimeUtc: string;
   dataFetchedAtTimeUtc: string;
 };
 
 const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
-  const { aucklandDhbsPopulationDoseData } = props;
-
-  const sortedAucklandDhbsPopulationDoseData =
-    aucklandDhbsPopulationDoseData.sort((a, b) =>
-      a.dhbName.localeCompare(b.dhbName)
-    );
```

Comment:
🙋  Do we want to keep alpha sort?

Reply:
Remember @finnclark mentioning the intent was to be North => South direction